tests/integ/sagemaker/serve/test_serve_mlflow_tensorflow_flavor_happy.py

```diff
@@ -0,0 +1,180 @@
+    input_data, output_data = test_data
+    return SchemaBuilder(
+        sample_input=input_data,
+        sample_output=output_data,
+        input_translator=custom_request_translator,
+        output_translator=custom_response_translator,
+    )
+
+
+@pytest.mark.skipif(
+    PYTHON_VERSION_IS_NOT_310,
+    reason="The goal of these test are to test the serving components of our feature",
+)
+def test_happy_tensorflow_sagemaker_endpoint_with_tensorflow_serving(
+    sagemaker_session,
+    tensorflow_schema_builder,
+    cpu_instance_type,
+    test_data,
+):
+    logger.info("Running in SAGEMAKER_ENDPOINT mode...")
+    caught_ex = None
+
+    iam_client = sagemaker_session.boto_session.client("iam")
+    role_arn = iam_client.get_role(RoleName=ROLE_NAME)["Role"]["Arn"]
+
+    model_artifacts_uri = "s3://{}/{}/{}/{}".format(
+        sagemaker_session.default_bucket(),
+        "model_builder_integ_test",
+        "mlflow",
+        "tensorflow",
+    )
+
+    model_path = S3Uploader.upload(
+        local_path=TENSORFLOW_MLFLOW_RESOURCE_DIR,
+        desired_s3_uri=model_artifacts_uri,
+        sagemaker_session=sagemaker_session,
+    )
+
+    model_builder = ModelBuilder(
+        mode=Mode.SAGEMAKER_ENDPOINT,
+        schema_builder=tensorflow_schema_builder,
+        role_arn=role_arn,
+        sagemaker_session=sagemaker_session,
+        model_metadata={"MLFLOW_MODEL_PATH": model_path},
+    )
+
+    model = model_builder.build(sagemaker_session=sagemaker_session)
+
+    with timeout(minutes=SERVE_SAGEMAKER_ENDPOINT_TIMEOUT):
+        try:
+            test_x, _ = test_data
+            logger.info("Deploying and predicting in SAGEMAKER_ENDPOINT mode...")
+            predictor = model.deploy(instance_type=cpu_instance_type, initial_instance_count=1)
+            logger.info("Endpoint successfully deployed.")
+            predictor.predict(test_x)
+            model_data_artifact = None
+            for artifact in Artifact.list(
+                source_uri=model_builder.s3_upload_path, sagemaker_session=sagemaker_session
+            ):
+                model_data_artifact = artifact
+            for association in Association.list(
+                destination_arn=model_data_artifact.artifact_arn,
+                sagemaker_session=sagemaker_session,
+            ):
+                assert (
+                    association.source_type == MODEL_BUILDER_MLFLOW_MODEL_PATH_LINEAGE_ARTIFACT_TYPE
+                )
+                break
+
+        except Exception as e:
+            caught_ex = e
+        finally:
+            cleanup_model_resources(
+                sagemaker_session=model_builder.sagemaker_session,
+                model_name=model.name,
+                endpoint_name=model.endpoint_name,
+            )
+            if caught_ex:
+                logger.exception(caught_ex)
+                ignore_if_worker_dies = "Worker died." in str(caught_ex)
```

Comment:
Why is the worker dying here? I had disabled it due to the accelerate bug in the past

Reply:
Don't know, was just copying from previous integ tests.

Reply:
Remove it please